recipes/2-Workflows/feature_importance/feature_importance.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
In general for this workflow - do we want to include this here as a use case since we already have an online doc for it?